sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -1307,7 +1307,7 @@ static AnalyzeSentimentResultCollection getAnalyzeSentimentResultCollectionForPa
                 false, TIME_NOW, getExtractKeyPhrasesResultCollectionForPagination(startIndex, firstPage), null))),
             IterableStream.of(asList(getExpectedAnalyzeSentimentActionResult(
                 false, TIME_NOW, getAnalyzeSentimentResultCollectionForPagination(startIndex, firstPage), null))),
-            null
+            IterableStream.of(Collections.emptyList())
```

Comment:
what is this param for?

Reply:
Extractive summarization task

Reply:
For better customer experience, can't the null -> empty list be handled by the client?
It isn't very intuitive to the user as to why they should prefer to send an empty list instead of the null.

Have we mentioned in the javadocs for a non-null requirement of this task?

Reply:
We are generating the expected result. Which user would expect a non-null list.

There is no user actions here.

Reply:
So this is the output result, then why do we need to re-record?